### ISSUE TITLE
LOGIN FAILED! (invalid response)

### ISSUE THREAD

**Issue by Unkn0wn-MDCLXIV** [sunsettrack4/telerising#15]
I’m getting the following error since a few days.
Has been working flawlessly for months.

`INFO - Unable to parse apptoken json data (trying old apptoken method now...)`
`ERROR - LOGIN FAILED! (invalid response)`
`ERROR - RESPONSE:`
`{"internal_code": 111, "http_status": 403, "success": false}`
`==== API PROCESS STOPPED! ==== `

**Comment by Tobi823** [comment]
I can confirm this bug - suddenly telerising does not work anymore (but it was working on 05.09.2020 with the perl locale warning)

I tried multiple zattoo.com accounts and i tried multiple servers with no success:

```
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = "",
        LC_ALL = (unset),
        LC_ADDRESS = "de_DE.UTF-8",
        LC_NAME = "de_DE.UTF-8",
        LC_MONETARY = "de_DE.UTF-8",
        LC_PAPER = "de_DE.UTF-8",
        LC_IDENTIFICATION = "de_DE.UTF-8",
        LC_TELEPHONE = "de_DE.UTF-8",
        LC_MEASUREMENT = "de_DE.UTF-8",
        LC_TIME = "de_DE.UTF-8",
        LC_NUMERIC = "de_DE.UTF-8",
        LANG = "en_DE.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").

 =========================                     I             +        
 TELERISING API v0.3.8                          I    I         +        
 =========================                       I  I       +      +    
                                                  II                    
ZZZZZZZZZ       AA     TTTTTTTTTT TTTTTTTTTT    888888        888888    
      ZZ        AA         TT         TT      88      88    88      88  
     ZZ        A  A        TT         TT     88        88  88        88 
    ZZ        AA  AA       TT         TT    88           888          88
   ZZ        AAAAAAAA      TT         TT    88           888          88
  ZZ        AA      AA     TT         TT     88        88  88        88 
 ZZ         AA      AA     TT         TT      88      88    88      88  
ZZZZZZZZZ  AA        AA    TT         TT        888888        888888    
(c) 2019-2020 Jan-Luca Neumann (sunsettrack4)
Please donate to support my work: https://paypal.me/sunsettrack4
==== API CONFIGURATION ====
INFO - Custom IP address or domain "127.0.0.1" will be used.
INFO - Custom Zattoo server "fr5-0" will be used.
INFO - Use custom ffmpeg library path "/usr/bin/ffmpeg"
INFO - Custom port "8080" will be used.
INFO - Youth protection pin will be used to request channel playlists.
INFO - YOUR DEFAULT SETTINGS
PLATFORM:   hls5
BANDWIDTH:  5000
PROFILE:    1
LOGLEVEL:   normal
IGN_MAX:    false
INFO - Unable to parse apptoken json data (trying old apptoken method now...)
ERROR - LOGIN FAILED! (invalid response)
ERROR - RESPONSE:
{"internal_code": 111, "http_status": 403, "success": false}
==== API PROCESS STOPPED! ====
```

**Comment by sunsettrack4** [comment]
Fixed 24e96621baa51b55b538e38e1e24f706112e8051

**Comment by Tobi823** [comment]
It works again - thanks for your fast response and your hard work :+1:
